docs/content/en/docs/getting-started/_index.md

```diff
@@ -331,10 +327,10 @@ Create some Keptn Grafana dashboards that will be available when Grafana is inst
 <!---x-release-please-start-version-->
```

Comment:
let's add this file here: https://github.com/keptn/lifecycle-toolkit/blob/main/release-please-config.json#L23

Reply:
nvm you already did that :D